RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt

```diff
@@ -0,0 +1,8 @@
+# |eta| min   |eta| max   effective area
```

Comment:
please add some documentation comments similar to effAreaElectrons_cone03_pfNeuHadronsAndPhotons_92X

Reply:
done